README.md

```diff
@@ -0,0 +1,35 @@
+# omer
+
+A webserver for helping Jews fulfill the mitzvah of counting the 49 days of the Omer between Pesach and Shavuos.
+
+During the season of use, it is hosted at [yossi.pythonanywhere.com](https://yossi.pythonanywhere.com).
+
+## Running locally
+
+To setup the local environment, you will need pyenv, or a system with python 3.10 already installed. (As of 2023-04-16, python 3.11 gives errors when installing the requirements.)
```

Comment:
3.11 doesnt work because numba is having difficulties getting 3.11 support out the door. Workarounds include removing the reference to numba in in requirements.txt, or installing each package in requirements.txt individually (skipping numba but not timezonefinder), or simply using python3.10.

pythonanywhere doesnt offer 3.11 yet anyway.